services/web/server/src/simcore_service_webserver/projects/_crud_api_delete.py

```diff
@@ -56,7 +57,12 @@ async def mark_project_as_deleted(
     """
     # NOTE: https://github.com/ITISFoundation/osparc-issues/issues/468
     db: ProjectDBAPI = ProjectDBAPI.get_from_app_context(app)
-    await db.check_delete_project_permission(user_id, f"{project_uuid}")
+    product_name = await db.get_project_product(project_uuid=project_uuid)
+    prj_access_rights = await get_user_project_access_rights(
+        app, project_id=project_uuid, user_id=user_id, product_name=product_name
+    )
+    if prj_access_rights.delete is False:
```

Comment:
minor: all these calls could be something like ```can_delete``` instead. same for write and read